x-pack/plugins/alerting/server/saved_objects/migrations.ts

```diff
@@ -859,6 +861,30 @@ function addMappedParams(
   return doc;
 }
 
+function convertSnoozes(
```

Comment:
Note that this function doesn't do anything with the new RRULE support. It's just a simple migration of existing non-recurring snoozes into the new model.